server/webapp/WEB-INF/rails.new/spec/controllers/api_v1/admin/environments_controller_spec.rb

```diff
@@ -245,29 +348,29 @@
         after :each do
           Rack::MockRequest::DEFAULT_ENV = {}
         end
-        it 'should route to update action of environments controller for alphanumeric environment name' do
-          expect(:patch => 'api/admin/environments/foo123').to route_to(action: 'update', controller: 'api_v1/admin/environments', name: 'foo123')
+        it 'should route to put action of environments controller for alphanumeric environment name' do
+          expect(:patch => 'api/admin/environments/foo123').to route_to(action: 'patch', controller: 'api_v1/admin/environments', name: 'foo123')
```

Comment:
What the test does is incorrect wrt to what the test says, for some of the the routing tests below